exporter/exporter.go

```diff
@@ -22,6 +22,8 @@ type Logs = internal.Logs
 // Settings configures exporter creators.
 type Settings = internal.Settings
 
+type Request = internal.Request
```

Comment:
Request is a concept exclusive to the exporterhelper, right? I don't think it should be exported in `exporter`. Why not just do this re-exporting in exporterhelper?

Reply:
That makes sense. Thanks for the feedback!
I was trying to follow the pattern of the re-exporting things in `exporter` but I agree having it in `exporterhelper` makes much sense in this case.